config.json

```diff
@@ -293,7 +297,7 @@
         },
         {
           "name": "network",
-          "viewName": "VPC Network Name *",
```

Comment:
Shouldn't this one receive `required` property?

Reply:
No. It is useful for narrowing the autocomplete search for subnets, but left blank the method still works.